### ISSUE TITLE
Docs: My kingdom for a basic ELI5 full explanation for next/image and next/future/image props

### ISSUE THREAD

**Issue by lack-of-gravitas** [vercel/next.js#40668]
### What is the improvement or update you wish to see?

Barebones documentations of core nextjs features don't help with adoption or use -- especially for new users of nextjs.

### Is there any context that might help us understand?

Nextjs own documentation on next/image and next/future/image is woeful and doesn't explain styling impacts properly and just provides a few barebones examples and expects the new adopter to figure the rest out..... where one goes to figure that out is unknown.

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/api-reference/next/future/image

**Comment by balazsorban44** [comment]
Thanks for the feedback, I forwarded it to the rest of the team. We are constantly trying to improve our documentation. If you have any specific things you want to see in the docs, please share them with us! :pray:
